client/package.json

```diff
@@ -4,45 +4,47 @@
   "private": true,
   "dependencies": {
     "@auth0/auth0-react": "^1.10.1",
-    "@date-io/date-fns": "^1.3.13",
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
-    "@headlessui/react": "^1.6.5",
     "@heroicons/react": "^1.0.6",
-    "@material-ui/core": "^5.0.0-beta.5",
-    "@material-ui/icons": "^5.0.0-beta.1",
-    "@material-ui/lab": "^5.0.0-alpha.44",
-    "@mui/icons-material": "^5.4.2",
     "@mui/lab": "^5.0.0-alpha.66",
     "@mui/material": "^5.3.0",
-    "@mui/styles": "^5.7.0",
     "@mui/x-data-grid": "^5.0.0-beta.5",
+    "@reduxjs/toolkit": "^1.8.3",
+    "@stripe/stripe-js": "^1.34.0",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
-    "@types/material-ui": "^0.21.12",
     "@types/node": "^16.11.41",
-    "@types/react": "^17.0.0",
+    "@types/react": "^17.0.47",
     "@types/react-dom": "^18.0.5",
     "axios": "^0.25.0",
     "daisyui": "^2.18.0",
     "date-fns": "^2.28.0",
-    "materialui-daterange-picker": "^1.1.92",
+    "deep-diff": "^1.0.2",
+    "eslint-import-resolver-typescript": "^3.3.0",
+    "final-form-arrays": "^3.0.2",
+    "flowbite": "^1.5.1",
+    "flowbite-react": "^0.1.3",
     "react": "^17.0.2",
     "react-datepicker": "^4.8.0",
     "react-dom": "^17.0.2",
+    "react-final-form-arrays": "^3.1.3",
     "react-loading": "^2.0.3",
+    "react-redux": "^8.0.2",
     "react-router": "^6.2.1",
     "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
     "react-scroll": "^1.8.7",
     "react-tailwind-table": "^1.1.1",
+    "redux-mock-store": "^1.5.4",
+    "redux-thunk": "^2.4.1",
     "stripe": "^9.9.0",
     "web-vitals": "^2.1.3"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "HTTPS=true SSL_CRT_FILE=localhost.pem react-scripts start",
```

Comment:
It looks like this may just be a problem with windows not recognizing these variables? I was able to start the client on windows after changing this line to "start": "set HTTPS=true set SSL_CRT_FILE=localhost.pem && react-scripts start",

Reply:
Wasn't an issue with the docker setup though

Reply:
Ah yes. Windows is a problematic OS for any cross platform development. I’ll install cross-env really quick and that should fix it.